navigation/Home/HomeTabNavigator.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react-native/no-unused-styles */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react/prop-types */
+/* eslint-disable react-native/no-inline-styles */
+/* eslint-disable react-native/no-color-literals */
```

Comment:
lol 🍭 eslint is driving you crazy as far as I can see.

Reply:
fixed